radar/infrastructure_config/zookeeper.yml

```diff
@@ -0,0 +1,18 @@
+version: 1
+name: ZooKeeper
+blip:
+  - version: "2.6"
+    ring: ASSESS
+description: |
+  [ZooKeeper](https://zookeeper.apache.org/) is a centralized service for maintaining configuration
+  information, naming, providing distributed synchronization, and providing group services.
+rationale:
+  We need to take a closer look into how to store and manage configuration etc.
+  ZooKeeper is one of the options.
+related:
+  - name: Consul
+    file: consul.html
+  - name: etcd
+    file: etcd.html
+  - name: Kafka
+    file: kafka.html
```

Comment:
I agree Etcd and Consul seem like better options, but since there is an initiative to implement Kafka, zookeepers are needed. The blip of ZooKeeper was added to keep the option open.

Reply:
We use ZooKeeper in prod but I don't think we really have an opinon of what is best.